presto-hive-hadoop2/bin/run_hive_s3_tests.sh

```diff
@@ -16,7 +16,9 @@ exec_in_hadoop_master_container sed -i \
  /etc/hadoop/conf/core-site.xml
 
 # create test table
-exec_in_hadoop_master_container /usr/bin/hive -e "CREATE EXTERNAL TABLE presto_test_s3(t_bigint bigint) LOCATION 's3a://${S3_BUCKET}/presto_test_s3/'"
+exec_in_hadoop_master_container hadoop fs -mkdir -p "s3a://${S3_BUCKET}/presto_test_external_fs/"
+exec_in_hadoop_master_container hadoop fs -copyFromLocal -f /tmp/test1.csv "s3a://${S3_BUCKET}/presto_test_external_fs/test1.csv"
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/dev/Introduction.html
> Updates to a single key are atomic. For example, if you PUT to an existing key, a subsequent read might return the old data or the updated data, but it will never return corrupted or partial data.